ovos_persona_server/ollama.py

```
@@ -371,3 +372,150 @@ async def tags(persona: Persona = Depends(get_default_persona)) -> OllamaTagsRes
         models=[model]
     )
 
+
+@ollama_router.get("/show")
+async def show(persona: Persona = Depends(get_default_persona)) -> JSONResponse:
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)